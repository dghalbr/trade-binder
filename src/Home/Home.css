.Home-center {
    margin: auto;
    display: block !important;
    text-align: center;
}