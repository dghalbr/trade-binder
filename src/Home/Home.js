import React, { Component } from 'react';

export default class Home extends Component {
  render() {
    return <div>Trade Binder Home Content</div>;
  }
}
