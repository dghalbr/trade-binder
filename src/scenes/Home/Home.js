import React, { Component } from 'react';
import { Page } from '../../components/Page/Page';

export class Home extends Component {
  render() {
    return <Page title="Home" />;
  }
}
