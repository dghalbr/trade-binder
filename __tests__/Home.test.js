import React from 'react';
import ReactDOM from 'react-dom';
import { BrowserRouter as Router } from 'react-router-dom';
import { Home } from '../src/scenes/Home/Home';
import renderer from 'react-test-renderer';
import toJson from 'enzyme-to-json';
import { shallow } from 'enzyme';

describe('Component: Home', () => {
    it('should match its empty snapshot', () => {
        const tree = renderer.create(
            <Home />
        ).toJSON();

        expect(tree).toMatchSnapshot();
    });
});