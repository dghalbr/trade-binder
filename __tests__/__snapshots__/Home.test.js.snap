// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: Home should match its empty snapshot 1`] = `
<div>
  Trade Binder Home Content
</div>
`;
